topo_processor/metadata/metadata_loaders/metadata_loader_imagery_historic.py

```diff
@@ -78,17 +80,34 @@ def populate_item(self, metadata_row, asset: Asset = None) -> None:
 
         collection.license = "CC-BY-4.0"
         collection.description = "Historical Imagery"
+        collection.extra_fields.update(
+            {
+                "linz:lifecycle": "completed",
+                "linz:history": "LINZ and its predecessors, Lands & Survey and Department of Survey and Land Information (DOSLI), commissioned aerial photography for the Crown between 1936 and 2008.\nOne of the predominant uses of the aerial photography at the time was the photogrammetric mapping of New Zealand, initially at 1inch to 1mile followed by the NZMS 260 and Topo50 map series at 1:50,000.\nThese photographs were scanned through the Crown Aerial Film Archive scanning project.",
+                # TODO: [TDE-237] to generate release versioning
+                "processing:software": {"Topo Processor": "0.1.0"},
+                # TODO: decision to be made on version ref comments [TDE-230] hardcode to '1' for now
+                "version": "1",
+            }
+        )
+
+        collection.add_extension(stac.StacExtensions.quality.value)
+
+        collection.add_linz_provider(LinzProviders.LTTW.value)
+        collection.add_linz_provider(LinzProviders.LMPP.value)
         collection.add_provider(Providers.NZAM.value)
 
         item.properties.update(
             {
                 "mission": title,
                 "platform": "Fixed-wing Aircraft",
                 "instruments": [metadata_row["camera"]],
-                "linz:photo_type": metadata_row["photo_type"],  # to be replaced by Linz:geospatial_type
+                "processing:software": {"Topo Processor": "0.1.0"},
```

Comment:
```python
>>> import pkg_resources
>>> my_version = pkg_resources.get_distribution('topo-processor').version
```
Might return the version number from `pyproject.toml`

Reply:
This has been split out to [TDE-237](https://linzgovt.atlassian.net/browse/TDE-237)